index.html

```diff
@@ -109,6 +110,7 @@ <h1 color="#FFFFFF" class="css-1rvqtgb">City of Boulder Careers</h1>
                                         data-uxi-widget-type="inputField" data-uxi-query-id=""
                                         data-uxi-number-job-results="18" class="css-yslldf">
                                         <div class="css-4zlbsc">
+                                            <label for="keywordSearchInput">Jobs or keywords:</label>
```

Comment:
Input now has a label which is used by assistive technologies such as screen readers to describe the purpose of the input element when it receives focus.

![Screen Shot 2024-11-23 at 11 40 10 PM](https://github.com/user-attachments/assets/1466749a-0639-4453-ab8c-e603d27b8c7b)